bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/listeners/MesgListener_Record.java

```diff
@@ -307,6 +311,11 @@ private void setRecord(final RecordMesg mesg) {
     */
    private void setRecord_DeveloperData(final RecordMesg mesg, final TimeData timeData) {
 
+      final boolean containsMultiplePowerDataSources =
```

Comment:
`containsMultiplePowerDataSources` should be move to the place where it is used -> below line 358, otherwise it would be computed for any case